resources/zib/zib-FamilySituation.xml

```diff
@@ -0,0 +1,275 @@
+      <path value="Observation.component.value[x]" />
+      <short value="CareResponsibility" />
+      <definition value="The activities the patient carries out to care for a dependent family member." />
+      <alias value="Zorgtaak" />
+      <type>
+        <code value="string" />
+      </type>
+      <mapping>
+        <identity value="zib-familysituation-v3.2-2020EN" />
+        <map value="NL-CM:7.13.10" />
+        <comment value="CareResponsibility" />
+      </mapping>
+    </element>
+    <element id="Observation.component:child">
+      <path value="Observation.component" />
+      <sliceName value="child" />
+    </element>
+    <element id="Observation.component:child.extension">
+      <path value="Observation.component.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="0" />
+    </element>
+    <element id="Observation.component:child.extension:livingAtHome">
+      <path value="Observation.component.extension" />
+      <sliceName value="livingAtHome" />
+      <min value="0" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-FamiliySituation.LivingAtHome" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Observation.component:child.extension:commentChild">
+      <path value="Observation.component.extension" />
+      <sliceName value="commentChild" />
+      <min value="0" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Comment" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Observation.component:child.extension:commentChild.value[x]">
+      <path value="Observation.component.extension.value[x]" />
+      <short value="CommentChild" />
+      <definition value="A comment on the childs family situation." />
+      <alias value="ToelichtingKind" />
+      <mapping>
+        <identity value="zib-familysituation-v3.2-2020EN" />
+        <map value="NL-CM:7.13.11" />
+        <comment value="CommentChild" />
+      </mapping>
+    </element>
+    <element id="Observation.component:child.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="67822003" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:child.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="DateOfBirth" />
+      <definition value="The child’s date of birth." />
+      <alias value="Geboortedatum" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
```

Comment:
Is this mapping really justified? The corresponding `Observation.component:child.code` is _67822003_, which indicates the child as a whole; here the `value` only reflects the birth date, so I'd expect LOINC code _2112-8_ (or even better a more specific one) as the accompanying `.code`. Wouldn't it conceptually be better to have a `.component` only indicating the child, with an extension to represent the birth date (even though I agree that another extension is not ideal)?